scripts/in_container/bin/run_tmux

```diff
@@ -53,24 +53,28 @@ tmux set mouse on
 tmux rename-window -t 0 'Main'
 tmux send-keys -t 'Main' 'bash' C-m 'clear' C-m
 
+
+COMMAND_PREFIX=""
+COMMAND_SUFFIX=""
+if [[ " ${INTEGRATIONS[*]} " =~ " jaegger " ]]; then
+    COMMAND_PREFIX="opentelemetry-instrument"
+    COMMAND_SUFFIX="--opentelemetry"
+fi
+
 tmux split-window -v
 tmux select-pane -t 1
-tmux send-keys '${COMMAND_PREFIX} airflow scheduler' C-m
+tmux send-keys '${COMMAND_PREFIX} airflow ${COMMAND_SUFFIX}' C-m
 
 tmux split-window -h
 tmux select-pane -t 2
-tmux send-keys '${COMMAND_PREFIX} airflow webserver' C-m
+tmux send-keys '${COMMAND_PREFIX} airflow webserver ${COMMAND_SUFFIX}' C-m
```

Comment:
I didn't see this before, It seems like I took out the scheduler part, I'll fix this and check